src/debug/Stream.h

```diff
@@ -207,6 +208,14 @@ void ResyncDebugLog(FILE *newDestination);
         } \
    } while (/*CONSTCOND*/ 0)
 
+/// Convenience wrapper for displaying protocol message flow in cache.log
+#define protocolTrace(SECTION, LABEL, ID, MESSAGE) \
```

Comment:
I suggest calling the new debugs() wrapper debugProtoHeader(), debugHeader(), debugProtoMetadata(). Keeping the same prefix for both macros will help with code understanding and maintenance.

If you are violently against that suggestion, then at least call the new function using a function-like name -- traceProtocol() -- instead of the proposed class/object-like name -- protocolTrace().

Reply:
Done in [65b9b1c](https://github.com/squid-cache/squid/pull/1105/commits/65b9b1c6cfaaafead9611aaa647d767f9734a4f1)